### PR TITLE
DD-5 Create a new ticket, DD-6 Initial API endpoint for Ticket

### PR DESCRIPTION
**Pull request by sopon007**
DD-006 Create a ticket and initial endpoint API

### CODE REVIEW

File: DevDynamo.Models/Ticket.cs [hunk 1]
```diff
@@ -7,18 +7,29 @@
...
 
-    [StringLength(50)]
-    public string Status { get; set; } = null!;
-  }
+        public string? Description { get; set; }
+
+        [StringLength(50)]
+        public string Status { get; set; } = null!;
+
+        public void Initial(string title, Guid projectid, string status)
```
Comment (surrealist) [inline DevDynamo.Models/Ticket.cs:28]: We might need to move this method to the constructor.

File: DevDynamo.Models/Ticket.cs [hunk 2]
```diff
@@ -7,18 +7,29 @@
...
+        [StringLength(100)]
+        public string Title { get; set; } = null!;
 
-    [StringLength(50)]
-    public string Status { get; set; } = null!;
-  }
+        public string? Description { get; set; }
+
+        [StringLength(50)]
+        public string Status { get; set; } = null!;
```
[the remark this answers is not in the document]
Comment (sopon007) [reply DevDynamo.Models/Ticket.cs:26]: Teacher krub. Can i have sample for change status is incorrect?

[the remark this answers is not in the document]
Comment (sopon007) [reply DevDynamo.Models/Ticket.cs:26]: I fixed code this below that not sure can enough for this case
image

**Comment by sopon007** [comment]
I have already fixed. Thank you krub.

**Review by surrealist — APPROVED** [review]

**Pull request merged**